autogluon_zeroshot/contexts/context_2023_03_19_bag_289.py

```diff
@@ -31,14 +32,18 @@ def load_context_2023_03_19_bag_289(
 
     path_bagged_root_s3_zs_input = f'{path_bagged_root_s3}zs_input/bagged_289/'
 
-    if load_from_local:
-        results_path = str(path_bagged_root / "608/results_ranked_valid.parquet")
-        results_by_dataset_path = str(path_bagged_root / "608/results_ranked_by_dataset_valid.parquet")
-        raw_path = str(path_bagged_root / "openml_ag_2023_03_19_zs_models.parquet")
-    else:
+    results_path = str(path_bagged_root / "608/results_ranked_valid.parquet")
+    results_by_dataset_path = str(path_bagged_root / "608/results_ranked_by_dataset_valid.parquet")
+    raw_path = str(path_bagged_root / "openml_ag_2023_03_19_zs_models.parquet")
+    path_automl = str(Paths.automl_289_results_root / "results_ranked_by_dataset_valid.csv")
+    local_files_exist = [Path(f).exists() for f in [results_path, results_by_dataset_path, raw_path, path_automl]]
+    if False in local_files_exist:
```

Comment:
Good catch, we could use a more pythonic way:
```suggestion
    local_files_exist = all(Path(f).exists() for f in [results_path, results_by_dataset_path, raw_path, path_automl])
    if not local_files_exist:
```